tutoriel/en/_posts/2016-12-21-install-nagios--and-snmp-plugins.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: post
+title: Install Nagios Plugins and SNMP scripts
+lang: en
+ref: nagios-plugins-snmp
+modified:
+description: How to install Nagios Plugins and SNMP scripts
+tags: [tutoriel, snmp, monitoring]
+image:
+  feature:
+  credit:
+  creditlink:
+author: algorys
+comments: true
+share:
+date: 2016-12-21T11:20:00+01:00
+---
+
+# Introduction
+
+In this tutorial, we'll install [Nagios Plugins](https://www.nagios.org/projects/nagios-plugins/) to get scripts who help to monitor hosts or services. We also install [Nagios SNMP from Manubulon](http://nagios.manubulon.com/) who give general snmp scripts.
+
```

Comment:
We should explain that manubulon scripts are quite old and not really maintained :/